app/app.go

```diff
@@ -349,6 +349,8 @@ func (app *App) Update(updateData App, w io.Writer) (err error) {
 	poolName := updateData.Pool
 	teamOwner := updateData.TeamOwner
 	routerName := updateData.Router
+	platform := updateData.Platform
+	updatePlatform := updateData.UpdatePlatform
```

Comment:
Do you really need this `updatePlatform` parameter? Couldn't you just set it to true when `plaform` param is set?